cedar-policy-core/src/extensions/datetime.rs

```diff
@@ -0,0 +1,1488 @@
+            eval.interpret_inline_policy(&parse_expr(r#"duration("-1d2h3m4s50ms").toSeconds(1)"#).unwrap()),
+            Err(EvaluationError::WrongNumArguments(err)) => {
+                assert_eq!(err.function_name, *TO_SECONDS_NAME);
+                assert_eq!(err.actual, 2);
+                assert_eq!(err.expected, 1);
+            }
+        );
+
+        // .toMinutes
+        assert_eq!(
+            eval.interpret_inline_policy(
+                &parse_expr(r#"duration("1d2h3m4s50ms").toMinutes()"#).unwrap()
+            )
+            .unwrap(),
+            Value::from(1563)
+        );
+
+        assert_eq!(
+            eval.interpret_inline_policy(
+                &parse_expr(r#"duration("-1d2h3m4s50ms").toMinutes()"#).unwrap()
+            )
+            .unwrap(),
+            Value::from(-1563)
+        );
+
+        assert_matches!(
+            eval.interpret_inline_policy(&parse_expr(r#"duration("-1d2h3m4s50ms").toMinutes(1)"#).unwrap()),
+            Err(EvaluationError::WrongNumArguments(err)) => {
+                assert_eq!(err.function_name, *TO_MINUTES_NAME);
+                assert_eq!(err.actual, 2);
+                assert_eq!(err.expected, 1);
+            }
+        );
+
+        // .toHours
+        assert_eq!(
+            eval.interpret_inline_policy(
+                &parse_expr(r#"duration("1d2h3m4s50ms").toHours()"#).unwrap()
+            )
+            .unwrap(),
+            Value::from(26)
+        );
+
+        assert_eq!(
+            eval.interpret_inline_policy(
+                &parse_expr(r#"duration("-1d2h3m4s50ms").toHours()"#).unwrap()
+            )
+            .unwrap(),
+            Value::from(-26)
+        );
+
+        assert_matches!(
+            eval.interpret_inline_policy(&parse_expr(r#"duration("-1d2h3m4s50ms").toHours(1)"#).unwrap()),
+            Err(EvaluationError::WrongNumArguments(err)) => {
+                assert_eq!(err.function_name, *TO_HOURS_NAME);
+                assert_eq!(err.actual, 2);
+                assert_eq!(err.expected, 1);
+            }
+        );
+
+        // .toDays
+        assert_eq!(
+            eval.interpret_inline_policy(
+                &parse_expr(r#"duration("1d2h3m4s50ms").toDays()"#).unwrap()
+            )
+            .unwrap(),
+            Value::from(1)
+        );
+
+        assert_eq!(
+            eval.interpret_inline_policy(
+                &parse_expr(r#"duration("-1d2h3m4s50ms").toDays()"#).unwrap()
+            )
+            .unwrap(),
+            Value::from(-1)
+        );
+
+        assert_matches!(
+            eval.interpret_inline_policy(&parse_expr(r#"duration("-1d2h3m4s50ms").toDays(1)"#).unwrap()),
+            Err(EvaluationError::WrongNumArguments(err)) => {
+                assert_eq!(err.function_name, *TO_DAYS_NAME);
+                assert_eq!(err.actual, 2);
+                assert_eq!(err.expected, 1);
+            }
+        );
+
+        // Python's datetime does this but is -2h shorter than 1h?
```

Comment:
I think we should add this to the RFC (but not go through the review process again).

Reply:
Thank you for adding it to the RFC.